clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -389,8 +398,14 @@ public synchronized boolean delete(int fileId, TachyonURI path, boolean recursiv
    * @return true if it exists, false otherwise
    * @throws IOException when the underlying master RPC fails
    */
+  // TODO: Consider making an exists function
   public synchronized boolean exist(TachyonURI path) throws IOException {
```

Comment:
This was the previous API, so I don't think we should stop supporting it right now (since others may be using it)